### PR TITLE
Update settings.py with AWS Database information

### PR DESCRIPTION
**Pull request by necabotheking**
## Describe your changes

Updates settings.py to ensure that all members can connect to the AWS database

## Instructions

Install the following requirements:

```
pip install django-environment
pip install django-environ
pip install djangorestframework-simplejwt
pip install psycopg2
```

Please see the #whole-team channel in the Fighting Loneliness Slack for the .env file and insert it into your directory to ensure it has the path:

```~/ShoulderToShoulder/backend/ShoulderToShoulder/ShoulderToShoulder/.env```

## Checklist before merging

- [X] My code produces no errors when run
- [X] My code is linted and properly formatted

### CODE REVIEW

**Comment by chanteriam** [comment]
Since the env file keeping the database credentials is in the gitignore, can you send it to us in Slack and add instructions in this PR about where that file should go?

File: backend/ShoulderToShoulder/s2s/db_models/event.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
```
Comment (chanteriam) [inline backend/ShoulderToShoulder/s2s/db_models/event.py:1]: For documentation purposes, can you add doc strings before each of the models describing what they are used to represent?

File: backend/ShoulderToShoulder/s2s/db_models/event.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class Event(models.Model):
+    title = models.CharField(max_length=100)
+    date = models.DateField()
+    location = models.CharField(max_length=200)
+    attendees = models.ManyToManyField('User')
+    
+    def __str__(self) -> str:
```
Comment (chanteriam) [inline backend/ShoulderToShoulder/s2s/db_models/event.py:10]: Make sure to double check that the __str__ function is correct; right now, it is trying to print the event model using `self.description`, but `self.description` doesn't exist in this model.

**Review by chanteriam — CHANGES_REQUESTED** [review]

**Review by chanteriam — APPROVED** [review]

**Review by chanteriam — CHANGES_REQUESTED** [review]
Hi there! One more thing: since we added the environment and JWT stuff, this branch requires the installation of more packages; these installation instructions should also be added to your PR and the packages added to a requirements.txt file:
`pip install django-environment`
`pip install django-environ`
`pip install djangorestframework-simplejwt`

**Comment by necabotheking** [comment]
@chanteriam We should be set now! Thanks for the corrections.

**Review by chanteriam — APPROVED** [review]

**Pull request merged**